packages/central-server/src/apiV2/feedItems/CreateFeedItems.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { BESAdminCreateHandler } from '../CreateHandler';
+import { uploadImage } from '../utilities';
+/**
+ * Handles POST endpoints:
+ * - /feedItems
+ */
+
+export class CreateFeedItems extends BESAdminCreateHandler {
+  async createRecord() {
+    const {
+      country_id,
+      permission_group_id,
+      creation_date,
+      template_variables: { title, image, body, link },
+      type,
+      geographical_area_id,
+      user_id,
+      record_id,
+    } = this.newRecordData;
+
+    const newFeedItem = await this.models.feedItem.create({
```

Comment:
Can you take a look into wrapping this in a transaction, similar to the way it happens in the projects handler? It'd be good here since we don't know if the image upload will succeed, so we'd prefer not to have a half baked feed item pointing to a broken image.